<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WayBillWidget</class>
 <widget class="QWidget" name="WayBillWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>563</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Накладная</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>13</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>13</height>
               </size>
              </property>
              <property name="text">
               <string>Номер</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="serialNumEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>108</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>108</width>
                <height>20</height>
               </size>
              </property>
              <property name="toolTip">
               <string comment="Номер акта"/>
              </property>
              <property name="toolTipDuration">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>13</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>13</height>
               </size>
              </property>
              <property name="text">
               <string>Дата</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>108</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>108</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>161</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="printExcelReport">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>20</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="text">
          <string>Вывод Excel файла</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Департамент</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>13</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>Название</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="departBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>152</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="newDepartBtn">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string comment="Добавить новый департамент"/>
            </property>
            <property name="toolTipDuration">
             <number>5</number>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>56</width>
                <height>13</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>56</width>
                <height>13</height>
               </size>
              </property>
              <property name="text">
               <string>Реквизиты</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>171</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="accesoriesEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>39</width>
                <height>13</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>39</width>
                <height>13</height>
               </size>
              </property>
              <property name="text">
               <string>Адрес</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>171</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="addressEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>27</width>
           <height>13</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>13</height>
          </size>
         </property>
         <property name="text">
          <string>ФИО</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="clientBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>13</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>13</height>
          </size>
         </property>
         <property name="text">
          <string>Телефон</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="telephoneEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>13</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>13</height>
          </size>
         </property>
         <property name="text">
          <string>E-mail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="emailEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolButton" name="newClientBtn">
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="Добавить нового представителя заказчика"/>
       </property>
       <property name="toolTipDuration">
        <number>5</number>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="editRowAction">
   <property name="text">
    <string>Изменить запись</string>
   </property>
   <property name="toolTip">
    <string>Edit Row</string>
   </property>
  </action>
  <action name="copyRowAction">
   <property name="text">
    <string>Копировать вставкой</string>
   </property>
   <property name="toolTip">
    <string>Copy Row</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="text">
    <string>Удалить запись</string>
   </property>
   <property name="toolTip">
    <string>Delete Row</string>
   </property>
  </action>
  <action name="insertRowAction">
   <property name="text">
    <string>Добавить запись</string>
   </property>
   <property name="toolTip">
    <string>Insert Row</string>
   </property>
  </action>
  <action name="updateTableAction">
   <property name="text">
    <string>Обновить таблицу</string>
   </property>
   <property name="toolTip">
    <string>Update Table</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
